assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * AccountCreate component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+
+// import Data from 'googlesitekit-data';
```

Comment:
The commented lines implement the data connection and can be uncommenting once that part is merged.

Reply:
Uncommented.